cmd/createorupdate/createorupdate.go

```diff
@@ -173,7 +173,7 @@ func (s *server) handlePut(w http.ResponseWriter, req *http.Request) {
 	s.write(oc)
 
 	// simulate Context with property bag
-	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Hour)
```

Comment:
@amanohar it would be useful to know the timeout used by the RP so we can use the same here. This is the timeout of the context passed to the plugins.